packages/shared/src/rpc/index.ts

```diff
@@ -146,8 +146,8 @@ const rename =
 const ingressEgressEnvironment = z
   .object({
     minimum_deposit_amounts: chainAssetMapFactory(numberOrHex, 0),
-    ingress_fees: chainAssetMapFactory(numberOrHex, 0),
-    egress_fees: chainAssetMapFactory(numberOrHex, 0),
+    ingress_fees: chainAssetMapFactory(numberOrHex.nullable(), null),
```

Comment:
why not default it to `0`?

Reply:
i think we want to throw an error in the `quote` endpoint if we dont know the fee (this should only happen if the pool has no liquidity anyway). when using null i can check against that in the quote endpoint to detect it 🙂

Reply:
it will be null if the backend cannot estimate it, i havent looked into when exactly that happens tbh. but i think its better to be on the safe side and not return a quote instead of returning a wrong one (without the fee) in this case